tests/admin_views/tests.py

```diff
@@ -6275,6 +6276,152 @@ def test_hidden_fields_small_window(self):
         finally:
             self.selenium.set_window_size(current_size["width"], current_size["height"])
 
+    def test_updating_related_objects_updates_fk_selects(self):
+        from selenium.webdriver.common.by import By
+        from selenium.webdriver.support.ui import Select
+
+        BORN_COUNTRY_SELECT_ID = "id_born_country"
+        LIVING_COUNTRY_SELECT_ID = "id_living_country"
+        FAVORITE_COUNTRY_TO_VACATION_SELECT_ID = "id_favorite_country_to_vacation"
+        CONTINENT_SELECT_ID = "id_continent"
+
+        def _get_HTML_inside_element_by_id(id_):
+            return self.selenium.find_element(By.ID, id_).get_attribute("innerHTML")
+
+        self.admin_login(
+            username="super", password="secret", login_url=reverse("admin:index")
+        )
+        add_url = reverse("admin:admin_views_traveler_add")
+        self.selenium.get(self.live_server_url + add_url)
+
+        # Add new Country from the BORN select
```

Comment:
Why `BORN` is uppercased? :thinking: 
```suggestion
        # Add new Country from the born_country select.
```

Reply:
Just to denote it was the "born select" but put the field it's better, thank you!